packages/adblocker/src/preprocessor.ts

```diff
@@ -0,0 +1,241 @@
+import { StaticDataView, sizeOfUTF8 } from './data-view';
+import { fastStartsWith } from './utils';
+
+export type EnvKeys =
```

Comment:
@seia-soto Could you confirm if we need this enumeration here. Since the type includes `string` these are mostly used as documentation?

Reply:
These are only for type hinting. If we need to set default values like `true` and `false`, we may define them here too.

Reply:
Yes, it means the autocomplete in editors. It will not add any extra type-safety as expected.